pkgs/by-name/re/responsively-desktop/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
```

Comment:
```suggestion

```
Unused

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/307444/commits/f4e58eef423896cbc497b7d5abdc14ae6284c7ee